client/src/pages/ManageBookings/NextBooking.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { styled } from '@mui/material/styles';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import CardContent from '@mui/material/CardContent';
+import CardActions from '@mui/material/CardActions';
+import { IconButton, Typography } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings'
+
+import useStyles from './useStyles';
+
+import Avatar from '@mui/material/Avatar';
+
+
+
+
+const NextBooking = () => {
+    const classes = useStyles()
+    return (
+        <div>
```

Comment:
So, thinking in React. This probably isn't the best approach, the current bookings and next booking style will be **identical** so we should just make one component but then change the props for each booking but the styling stays the same so we can have this be more maintainable and keep the codebase concise.